gflownet.py

```diff
@@ -1238,13 +1262,21 @@ def logq(path_list, actions_list, model, env):
     # TODO: this method is probably suboptimal, since it may repeat forward calls for
     # the same nodes.
     log_q = torch.tensor(1.0)
+    loginf = 1e3
     for path, actions in zip(path_list, actions_list):
         path = path[::-1]
         actions = actions[::-1]
         path_obs = np.asarray([env.state2obs(state) for state in path])
+        done = [0] * len(path)
+        masks = tf(
+            [
+                env.get_mask_invalid_actions(path[idx], done[idx])
```

Comment:
Why do we need to pass `done[idx]`?

Reply:
The function prototype is `get_mask_invalid_actions(self, state=None, done=None)` in the `grid` and `aptamers`. I just noticed it is different in the abstract method declaration in class `GFlowNetEnv`

Reply:
I changed it in class `GflownetEnv`